server/app.py

```diff
@@ -3,13 +3,31 @@
 # graphql playground
 from ariadne.constants import PLAYGROUND_HTML
 
+from flask_sqlalchemy import SQLAlchemy
+from models import *
+
 # resolvers
 from graph.resolvers import resolvers
 
 type_def = gql(load_schema_from_path("graph/schema/"))
 SCHEMA = make_executable_schema(type_def, resolvers)
 
 app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///users.sqlite3'   #The database URI that should be used for the connection
```

Comment:
we arent using sqlite so the url wouldnt really work, we could probably set it up in a config file and take the url from it

Reply:
That's a uri, it's used as a default db and works fine with local files. I say this because I came across a lot of examples that set up their database as I did in models.py and it still worked. Might be wrong here though in which case I can figure out what we're doing with this today.

Reply:
It's created through db.create_all apparently and then tables are made when you session.commit() everything.